shared/constants/swaps.js

```diff
@@ -74,3 +76,7 @@ export const SWAPS_CHAINID_DEFAULT_TOKEN_MAP = {
   [SWAPS_TESTNET_CHAIN_ID]: TEST_ETH_SWAPS_TOKEN_OBJECT,
   [BNB_CHAIN_ID]: BNB_SWAPS_TOKEN_OBJECT,
 };
+
+export const SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP = {
+  [BNB_CHAIN_ID]: BSC_DEFAULT_BLOCK_EXPLORER_URL,
```

Comment:
This default should probably not be swaps-specific. We could use this as a default custom block explorer everywhere. We can adjust that later though.

Reply:
We should really consolidate this link/label generation logic in a utility function or hook, so that it's easier to add more special cases like this everywhere at once.

Unless we wanted to consider making this block explorer the default in the network form itself, and adding it via a migration for anyone that already has this network set. :thinking:

Reply:
I created a ticket for this here https://github.com/MetaMask/metamask-extension/issues/10750